pkg/cmd/scan/output/console.go

```diff
@@ -35,13 +35,27 @@ func NewConsole() *Console {
 
 func (c *Console) Write(analysis *analyser.Analysis) error {
 	if analysis.Summary().TotalDeleted > 0 {
+		groupedBySource := make(map[string][]*resource.Resource)
+
+		for _, deletedResource := range analysis.Deleted() {
+			key := deletedResource.Source.Source()
+
+			if _, exist := groupedBySource[key]; !exist {
+				groupedBySource[key] = []*resource.Resource{deletedResource}
+				continue
+			}
+
+			groupedBySource[key] = append(groupedBySource[key], deletedResource)
+		}
+
 		fmt.Println("Found missing resources:")
-		deletedByType, keys := groupByType(analysis.Deleted())
-		for _, ty := range keys {
-			fmt.Printf("  %s:\n", ty)
-			for _, res := range deletedByType[ty] {
-				humanString := fmt.Sprintf("    - %s", res.TerraformId())
-				if humanAttrs := formatResourceAttributes(res); humanAttrs != "" {
+
+		for source, deletedResources := range groupedBySource {
+			fmt.Print(color.BlueString("  From %s:\n", source))
```

Comment:
To be consistent with actual behavior on changes, you can remove the `:` character.

`fmt.Print(color.BlueString("  From %s\n", source))`